sp800-63d/vot_mapping.md

```diff
@@ -118,25 +118,87 @@ FAL maps into the VoT component A using a numeric identifier. Only one numeric i
+
+**Example:** C2.Cx.Cv could be asserted if implementing AAL2 and an authenticator is used that provides verifier impersonation resistance, since verifier impersonation resistance is not required under AAL2.
+
+Any additional component categories SHALL NOT be used.
+
+### Identity Proofing Characteristics
+
+If implementing IAL2 and KBV is used for identity verification, Pk SHALL be indicated in the vector (e.g., P2.Pk).
+
+If using a trusted referee acting on behalf of the applicant in the identity proofing process, Pt SHALL be indicated in the vector.
+
+If the subscriber is identity proofed in-person, Pi SHOULD be indicated in the vector.
+
+If the subscriber is identity proofed remotely, Pr SHOULD be indicated in the vector.
+
+If confirming an applicant's address by sending an enrollment code to a postal address, Pa SHOULD be indicated in the vector.
+
+### Authenticator Types
+
+The authenticator type(s) used SHALL be indicated in the vector with the following exceptions.
+
+If implementing AAL3 and authentication is achieved using one of the following two options, the authenticator types used MAY NOT be indicated in the vector.
+
+* Multi-Factor Cryptographic Device; OR
+* Single-Factor Cryptographic Device used in conjunction with Memorized Secret
+
+If the user authenticates using a restricted authenticator, Cr and the value representing the authenticator used SHALL be indicated in the vector.
+
+**Example:** Cr.Co would be asserted if the user authenticates using a secret sent via SMS to an out-of-band device.
+
+### Authentication Characteristics
+
+If implementing AAL1 or AAL2, and:
+
+* the user authenticates with an authenticator that provides verifier impersonation resistance, Cv SHOULD be indicated in the vector.
+* the user authenticates with an authenticator that provides verifier compromise resistance, Cs SHOULD be indicated in the vector.
+* authentication intent from at least one authenticator is demonstrated, Cn SHOULD be indicated in the vector.
+* for non-government agency authenticators and verifiers, authenticator(s) or verifier(s) are validated to meet the requirements of FIPS 140 Level 1, Ci SHOULD be indicated in the vector.
+
+If using a biometric for authentication, and:
+
+* biometric comparison is performed centrally (not locally on the claimant's device), Ck SHALL be indicated in the vector.
+* the biometric system implements PAD, Ck SHOULD be indicated in the vector.
+
+**NOTE: Should we add a vector value for FMR?**
+
+MitM attack resistance
+
+**NOTE: I'm struggling to find the case where a MitM resistance value is interesting - it seems to be required most of the time and would be redundant with the xAL indicated in the vector.**
```

Comment:
I think it's interesting when used with C1

Reply:
@jricher My understanding is that C1 would imply MitM resistance since it's required at AAL1. Or what is the interesting aspect to C1 with this value as well?

Reply:
Ok, you're right -- I was thinking that it was a SHOULD at AAL1, but if it's required across all AAL's then we should remove this value.